index.html

```diff
@@ -0,0 +1,50 @@
+<html>
+  <head>
+    <script src="/dist/main.js" defer></script>
+    <style>
```

Comment:
If you're already statically serving files, you should serve these as a `.css` file.